app/services/organization_service.py

```
@@ -0,0 +1,432 @@
+        requesting_user_id: str,
+        target_user_id: str,
+    ) -> bool:
+        """Remove a member. Requires owner/admin. Owners cannot remove themselves."""
+        membership = await self._get_membership(org_id, requesting_user_id)
+        if not membership or membership.role not in ("owner", "admin"):
+            return False
+
+        # Prevent owner from removing themselves
+        target_membership = await self._get_membership(org_id, target_user_id)
+        if target_membership and target_membership.role == "owner" and requesting_user_id == target_user_id:
+            raise ValueError("Owner cannot remove themselves from the organization")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)